src/pages/careers/_model/_locations/_locations.ts

```diff
@@ -66,6 +67,7 @@ export const cyberjaya: LocationsType = {
 
 export const malta: LocationsType = {
     name: 'malta',
+    jobs_name: 'Birkirkara',
```

Comment:
As @mitra-deriv suggested, let's change the key names
```suggestion
    job_location: 'Birkirkara',
```